src/pcapi/core/offers/api.py

```diff
@@ -280,6 +287,13 @@ def as_utc_without_timezone(d: datetime.datetime) -> datetime.datetime:
     return stock
 
 
+def _invalidate_bookings(bookings: typing.List[Booking]) -> typing.List[Booking]:
```

Comment:
```suggestion
def _invalidate_bookings(bookings: List[Booking]) -> List[Booking]:
```

`List` est déjà importé.